source/CommonJobs/CommonJobs.Application.EvalForm/DTOs/UpdateEvaluationDTO.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CommonJobs.Application.EvalForm.DTOs
```

Comment:
C# convention is to use PascalCase for acronyms with more than 2 letters.

See the header  "Capitalization Rules for Acronyms" in https://msdn.microsoft.com/en-us/library/vstudio/ms229043%28v=vs.100%29.aspx (I hate the pages without anchor marks or ids in the titles):

> **Do capitalize only the first character of acronyms with three or more characters, except the first word of a camel-cased identifier.**
> A class named XmlWriter is an example of a long acronym used as the first word of a Pascal-cased identifier. A parameter named htmlReader is an example of a long acronym used as the first word of a camel-cased identifier.